Source/Resources/en.lproj/Localizable.strings

```diff
@@ -39,3 +39,4 @@
 "county" = "County";
 "postcode" = "Postcode";
 "phone" = "Phone";
+"Search" = "Search...";
```

Comment:
The key should be more descriptive of where it will be used. I'd recommend `CountrySearchPlaceholder`.

Also the three dots in a row ("...") is actually an ellipsis and there's a different unicode for it that's used generally ("…"). You can copy from here or use Mac `Option + ;` to write.